app/main/routes.py

```diff
@@ -211,6 +211,11 @@ def how_to_use():
     return render_template("how-to-use-this-service.html")
 
 
+@bp.route("/terms-of-use", methods=["GET"])
+def terms_of_use():
```

Comment:
Is this terms of use page defined yet or just a placeholder?

Just wondering if its worth testing anything in its content

Reply:
just placeholder for now @anthonyhashemi